### PR TITLE
fix adaption typo

### PR DESCRIPTION
**Pull request by 315930399**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1886/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1886/

File: chapter_attention-mechanisms/attention-scoring-functions.md [hunk 1]
```diff
@@ -471,7 +471,7 @@ class DotProductAttention(nn.Module):
     # Shape of `valid_lens`: (`batch_size`,) or (`batch_size`, no. of queries)
     def forward(self, queries, keys, values, valid_lens=None):
         d = queries.shape[-1]
-        # Set `transpose_b=True` to swap the last two dimensions of `keys`
+        # Set `keys.transpose(1,2)` to swap the last two dimensions of `keys`
```
Comment (AnirudhDagar) [inline chapter_attention-mechanisms/attention-scoring-functions.md:474]:
```suggestion
        # Swap the last two dimensions of `keys` with `keys.transpose(1,2)`
```

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
@315930399 nice catch! I've made a small comment please make the changes if that sounds better. :))

**Comment by mli** [comment]
Job d2l-en/PR-1886/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1886/

**Review by AnirudhDagar — APPROVED** [review]
Thanks! LGTM

**Pull request merged**